src/main/scala/org/scalacheck/ScalaCheckFramework.scala

```diff
@@ -17,7 +17,7 @@ import org.scalacheck.Test.matchPropFilter
 
 private abstract class ScalaCheckRunner extends Runner {
 
-  val args: Array[String]
+  val args: Array[String | Null]
```

Comment:
A Scala field overrides some Java field. We need to add |Null to the field type to make the override still work.